addons/website_sale/views/templates.xml

```diff
@@ -1507,6 +1493,66 @@
+            <div id="add_to_cart_wrap" t-attf-class="{{'d-none' if combination_info['prevent_zero_price_sale'] else 'd-flex'}} flex-grow-1 flex-lg-grow-0 flex-wrap align-items-center gap-2 {{'' if website.product_page_image_width == '50_pc' else ' w-100'}}">
+                <a data-animation-selector=".o_wsale_product_images" role="button" id="add_to_cart" t-attf-class="btn btn-primary js_check_product a-submit flex-grow-1" href="#">
+                    <i class="fa fa-shopping-cart me-2"/>
+                    Add to cart
+                </a>
+            </div>
+            <div id="product_option_block" t-att-class="'d-flex d-empty-none flex-grow-1 flex-lg-grow-0 flex-wrap gap-2' + ('' if website.product_page_image_width == '50_pc' else ' w-100')"/>
+        </div>
+    </template>
+
+    <template id="cta_wrapper_boxed" inherit_id="website_sale.cta_wrapper" active="False" name="Bordered CTA Wrapper">
+        <xpath expr="//div[@id='o_wsale_cta_wrapper']" position="attributes">
+            <attribute name="t-att-class" remove="pt-3" add="' o_wsale_cta_wrapper_boxed border rounded p-3'" separator=" + "/>
+        </xpath>
+        <xpath expr="//div[@id='add_to_cart_wrap']" position="attributes">
+            <attribute name="t-attf-class" remove="flex-lg-grow-0" separator=" "/>
+        </xpath>
+        <xpath expr="//div[@id='product_option_block']" position="attributes">
+            <attribute name="t-att-class" remove="flex-lg-grow-0" separator=" "/>
+        </xpath>
+        <xpath expr="//div[@id='o_wsale_cta_wrapper']" position="inside">
+            <div class="d-flex gap-5 flex-grow-1 justify-content-between align-items-center order-first w-100 mb-3 pb-3 border-bottom">
+                <span class="text-muted">Price</span>
+                <t t-call="website_sale.product_price"/>
+            </div>
+        </xpath>
+    </template>
+
+    <template id="cta_wrapper_large" inherit_id="website_sale.cta_wrapper" active="False" name="Large CTA Wrapper">
+        <xpath expr="//div[@id='o_wsale_cta_wrapper']" position="attributes">
+            <attribute name="t-att-class" add="' o_wsale_cta_wrapper_large'" separator=" + "/>
+        </xpath>
+        <xpath expr="//div[@id='add_to_cart_wrap']" position="attributes">
+            <attribute name="t-attf-class" remove="flex-lg-grow-0" separator=" "/>
+        </xpath>
+        <xpath expr="//div[@id='product_option_block']" position="attributes">
+            <attribute name="t-att-class" remove="flex-lg-grow-0" separator=" "/>
+        </xpath>
+        <xpath expr="//a[@id='add_to_cart']" position="attributes">
+            <attribute name="t-attf-class" add="btn-lg {{('' if website.product_page_image_width == '50_pc' else ' px-xl-0')}}" separator=" "/>
+        </xpath>
+    </template>
```

Comment:
I'm unsure but I think it's getting too complex, since `cta_wrapper` is a very small template, I think it could be clearer to replace it's content directly instead of tweaking the attributes. 

If we do this, `cta_separator` would have to xpath a variable instead of the attributes and you'll have to check the `product_quantity` template to ensure it still works.

However it's maybe overkill and factoring the conditions will already make it easier to read. I'll refer to your judgement